### ISSUE TITLE
rv64 make error

### ISSUE THREAD

**Issue by YanxingLiu** [OAID/Tengine#1253]
I want to build Tengine for allwinnertech D1 SBC. And I follow the instructions in this link https://zhuanlan.zhihu.com/p/368640131. But I got an error of
riscv64-unknown-linux-gnu-gcc: error: unrecognized command line option '-mfp16'.
![image](https://user-images.githubusercontent.com/42299757/149083056-0614946c-46a5-4ec2-b639-e23e805bc6e8.png)

I have tried to delete flag -mfp16 in toolchains/rv64-c906.toolchain.cmake, but it still did not work.

**Comment by YanxingLiu** [comment]
I want to build Tengine for allwinnertech D1 SBC. And I follow the instructions in this link https://zhuanlan.zhihu.com/p/368640131. But I got an error of 
riscv64-unknown-linux-gnu-gcc: error: unrecognized command line option '-mfp16'. 
![image](https://user-images.githubusercontent.com/42299757/149082768-c6d51ab8-355e-4c8e-a7e9-1c022cda508c.png)

I have tried to delete flag -mfp16 in toolchains/rv64-c906.toolchain.cmake, but it still did not work.
